.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Please sort into logical groups with comment headers. Sort groups in order of specificity.
+# For example, default owners should always be the first group.
+# Sort lines alphabetically within these groups to avoid accidentally adding duplicates.
+#
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Default file owners
+* @bitwarden/team-admin-console-dev
```

Comment:
ℹ️ @differsthecat thought AC might be the best assignment to start. We need some kind of default for maintenance to work.